sortfilterproxymodel.qbs

```diff
@@ -46,6 +47,7 @@ Product
             return defines
         }
 
+        cpp.linkerFlags: "--no-as-needed"
```

Comment:
> An alternative would be to create an actual qml plugin out of this as described here ( oKcerG#25 ), and then have something like:
>
> `engine.addPluginPath(QCoreApplication::applicationDirPath());`

@saidinesh5 This is the kind of thing I was thinking - wouldn't that be a "cleaner" solution? Shouldn't be all that difficult, right? :wink: